commit 5456af201d4c0a2950200ba756e5f8314ddbbccd
Author: Carlos O'Donell <carlos@redhat.com>
Date:   Thu May 11 14:00:41 2023 -0400

    stdio-common: Reformat Makefile.
    
    Reflow Makefile.
    Sort using scripts/sort-makefile-lines.py.
    
    Code generation is changed as routines are linked in sorted order
    as expected.
    
    No regressions on x86_64 and i686.
    
    (cherry picked from commit c3004417afc98585089a9282d1d4d60cdef5317a)
    (cherry picked from commit 5b4e90230b8147e273585bf296bf1a9fb6e2b4c2)

diff --git a/stdio-common/Makefile b/stdio-common/Makefile
index 803f16dae030cb72..76d1d0a1193aa109 100644
--- a/stdio-common/Makefile
+++ b/stdio-common/Makefile
@@ -22,82 +22,241 @@ subdir	:= stdio-common
 
 include ../Makeconfig
 
-headers	:= stdio_ext.h printf.h bits/printf-ldbl.h bits/stdio_lim.h
-
-routines	:=							      \
-	ctermid cuserid							      \
-	_itoa _itowa itoa-digits itoa-udigits itowa-digits		      \
-	vfprintf vprintf printf_fp reg-printf printf-prs printf_fphex	      \
-	reg-modifier reg-type						      \
-	printf_size fprintf printf snprintf sprintf asprintf dprintf	      \
-	vfwprintf vfscanf vfwscanf					      \
-	fscanf scanf sscanf						      \
-	perror psignal							      \
-	tmpfile tmpfile64 tmpnam tmpnam_r tempnam tempname		      \
-	getline getw putw						      \
-	remove rename renameat renameat2				      \
-	flockfile ftrylockfile funlockfile				      \
-	isoc99_scanf isoc99_vscanf isoc99_fscanf isoc99_vfscanf isoc99_sscanf \
-	isoc99_vsscanf							      \
-	psiginfo gentempfd						      \
-	vfscanf-internal vfwscanf-internal iovfscanf			      \
-	vfprintf-internal vfwprintf-internal
-
-aux	:= errlist siglist printf-parsemb printf-parsewc fxprintf
-
-tests := tstscanf test_rdwr test-popen tstgetln test-fseek \
-	 temptest tst-fileno test-fwrite tst-ungetc tst-ferror \
-	 xbug errnobug \
-	 bug1 bug2 bug3 bug4 bug5 bug6 bug7 bug8 bug9 bug10 bug11 bug12 bug13 \
-	 tfformat tiformat tllformat tstdiomisc tst-printfsz tst-wc-printf \
-	 scanf1 scanf2 scanf3 scanf4 scanf5 scanf7 scanf8 scanf9 scanf10 \
-	 scanf11 scanf12 tst-tmpnam tst-cookie tst-obprintf tst-sscanf \
-	 tst-swprintf tst-fseek tst-fmemopen test-vfprintf tst-gets \
-	 tst-perror tst-sprintf tst-rndseek tst-fdopen tst-fphex bug14 \
-	 tst-popen tst-unlockedio tst-fmemopen2 tst-put-error tst-fgets \
-	 tst-fwrite bug16 bug17 tst-swscanf tst-sprintf2 bug18 bug18a \
-	 bug19 bug19a tst-popen2 scanf13 scanf14 scanf15 bug20 bug21 bug22 \
-	 scanf16 scanf17 tst-setvbuf1 tst-grouping bug23 bug24 \
-	 bug-vfprintf-nargs tst-long-dbl-fphex tst-fphex-wide tst-sprintf3 \
-	 bug25 tst-printf-round bug23-2 bug23-3 bug23-4 bug26 tst-fmemopen3 \
-	 tst-printf-bz18872 tst-vfprintf-width-prec tst-fmemopen4 \
-	 tst-vfprintf-user-type \
-	 tst-vfprintf-mbs-prec \
-	 tst-scanf-round \
-	 tst-renameat2 tst-bz11319 tst-bz11319-fortify2 \
-	 scanf14a scanf16a \
-	 tst-printf-bz25691 \
-	 tst-vfprintf-width-prec-alloc \
-	 tst-printf-fp-free \
-	 tst-printf-fp-leak \
-	 test-strerr
-
-
-test-srcs = tst-unbputc tst-printf tst-printfsz-islongdouble
+headers := \
+  bits/printf-ldbl.h \
+  bits/stdio_lim.h \
+  printf.h \
+  stdio_ext.h \
+  # headers
+
+routines := \
+  _itoa \
+  _itowa \
+  asprintf \
+  ctermid \
+  cuserid \
+  dprintf \
+  flockfile \
+  fprintf \
+  fscanf \
+  ftrylockfile \
+  funlockfile \
+  gentempfd \
+  getline \
+  getw \
+  iovfscanf \
+  isoc99_fscanf \
+  isoc99_scanf \
+  isoc99_sscanf \
+  isoc99_vfscanf \
+  isoc99_vscanf \
+  isoc99_vsscanf \
+  itoa-digits \
+  itoa-udigits \
+  itowa-digits \
+  perror \
+  printf \
+  printf-prs \
+  printf_fp \
+  printf_fphex \
+  printf_size \
+  psiginfo \
+  psignal \
+  putw \
+  reg-modifier \
+  reg-printf \
+  reg-type \
+  remove \
+  rename \
+  renameat \
+  renameat2 \
+  scanf \
+  snprintf \
+  sprintf \
+  sscanf \
+  tempnam \
+  tempname \
+  tmpfile \
+  tmpfile64 tmpnam \
+  tmpnam_r \
+  vfprintf \
+  vfprintf-internal \
+  vfscanf \
+  vfscanf-internal \
+  vfwprintf \
+  vfwprintf-internal \
+  vfwscanf \
+  vfwscanf-internal \
+  vprintf \
+  # routines
+
+aux := \
+  errlist \
+  fxprintf \
+  printf-parsemb \
+  printf-parsewc \
+  siglist \
+  # aux
+
+tests := \
+  bug-vfprintf-nargs \
+  bug1 \
+  bug10 \
+  bug11 \
+  bug12 \
+  bug13 \
+  bug14 \
+  bug16 \
+  bug17 \
+  bug18 \
+  bug18a \
+  bug19 \
+  bug19a \
+  bug2 \
+  bug20 \
+  bug21 \
+  bug22 \
+  bug23 \
+  bug23-2 \
+  bug23-3 \
+  bug23-4 \
+  bug24 \
+  bug25 \
+  bug26 \
+  bug3 \
+  bug4 \
+  bug5 \
+  bug6 \
+  bug7 \
+  bug8 \
+  bug9 \
+  errnobug \
+  scanf1 \
+  scanf10 \
+  scanf11 \
+  scanf12 \
+  scanf13 \
+  scanf14 \
+  scanf14a \
+  scanf15 \
+  scanf16 \
+  scanf16a \
+  scanf17 \
+  scanf2 \
+  scanf3 \
+  scanf4 \
+  scanf5 \
+  scanf7 \
+  scanf8 \
+  scanf9 \
+  temptest \
+  test-fseek \
+  test-fwrite \
+  test-popen \
+  test-strerr \
+  test-vfprintf \
+  test_rdwr \
+  tfformat \
+  tiformat \
+  tllformat \
+  tst-bz11319 \
+  tst-bz11319-fortify2 \
+  tst-cookie \
+  tst-fdopen \
+  tst-ferror \
+  tst-fgets \
+  tst-fileno \
+  tst-fmemopen \
+  tst-fmemopen2 \
+  tst-fmemopen3 \
+  tst-fmemopen4 \
+  tst-fphex \
+  tst-fphex-wide \
+  tst-fseek \
+  tst-fwrite \
+  tst-gets \
+  tst-grouping \
+  tst-long-dbl-fphex \
+  tst-obprintf \
+  tst-perror \
+  tst-popen \
+  tst-popen2 \
+  tst-printf-bz18872 \
+  tst-printf-bz25691 \
+  tst-printf-fp-free \
+  tst-printf-fp-leak \
+  tst-printf-round \
+  tst-printfsz \
+  tst-put-error \
+  tst-renameat2 \
+  tst-rndseek \
+  tst-scanf-round \
+  tst-setvbuf1 \
+  tst-sprintf \
+  tst-sprintf2 \
+  tst-sprintf3 \
+  tst-sscanf \
+  tst-swprintf \
+  tst-swscanf \
+  tst-tmpnam \
+  tst-ungetc \
+  tst-unlockedio \
+  tst-vfprintf-mbs-prec \
+  tst-vfprintf-user-type \
+  tst-vfprintf-width-prec \
+  tst-vfprintf-width-prec-alloc \
+  tst-wc-printf \
+  tstdiomisc \
+  tstgetln \
+  tstscanf \
+  xbug \
+  # tests
+
+test-srcs = \
+  tst-printf \
+  tst-printfsz-islongdouble \
+  tst-unbputc \
+  # test-srcs
 
 ifeq ($(run-built-tests),yes)
-tests-special += $(objpfx)tst-unbputc.out $(objpfx)tst-printf.out \
-		 $(objpfx)tst-printf-bz18872-mem.out \
-		 $(objpfx)tst-setvbuf1-cmp.out \
-		 $(objpfx)tst-vfprintf-width-prec-mem.out \
-		 $(objpfx)tst-printfsz-islongdouble.out \
-		 $(objpfx)tst-printf-bz25691-mem.out \
-		 $(objpfx)tst-printf-fp-free-mem.out \
-		 $(objpfx)tst-printf-fp-leak-mem.out
-generated += tst-printf-bz18872.c tst-printf-bz18872.mtrace \
-	     tst-printf-bz18872-mem.out \
-	     tst-vfprintf-width-prec.mtrace tst-vfprintf-width-prec-mem.out \
-	     tst-printf-bz25691.mtrace tst-printf-bz25691-mem.out \
-	     tst-printf-fp-free.mtrace tst-printf-fp-free-mem.out \
-	     tst-printf-fp-leak.mtrace tst-printf-fp-leak-mem.out
-endif
+tests-special += \
+  $(objpfx)tst-printf-bz18872-mem.out \
+  $(objpfx)tst-printf-bz25691-mem.out \
+  $(objpfx)tst-printf-fp-free-mem.out \
+  $(objpfx)tst-printf-fp-leak-mem.out \
+  $(objpfx)tst-printf.out \
+  $(objpfx)tst-printfsz-islongdouble.out \
+  $(objpfx)tst-setvbuf1-cmp.out \
+  $(objpfx)tst-unbputc.out \
+  $(objpfx)tst-vfprintf-width-prec-mem.out \
+  # tests-special
+
+generated += \
+  tst-printf-bz18872-mem.out \
+  tst-printf-bz18872.c \
+  tst-printf-bz18872.mtrace \
+  tst-printf-bz25691-mem.out \
+  tst-printf-bz25691.mtrace \
+  tst-printf-fp-free-mem.out \
+  tst-printf-fp-free.mtrace \
+  tst-printf-fp-leak-mem.out \
+  tst-printf-fp-leak.mtrace \
+  tst-vfprintf-width-prec-mem.out \
+  tst-vfprintf-width-prec.mtrace \
+  # generated
+endif # $(run-built-tests)
 
 tests-special += $(objpfx)tst-errno-manual.out
 
 include ../Rules
 
 ifeq ($(run-built-tests),yes)
-LOCALES := de_DE.ISO-8859-1 de_DE.UTF-8 en_US.ISO-8859-1 ja_JP.EUC-JP
+LOCALES := \
+  de_DE.ISO-8859-1 \
+  de_DE.UTF-8 \
+  en_US.ISO-8859-1 \
+  ja_JP.EUC-JP \
+  # LOCALES
 include ../gen-locales.mk
 
 $(objpfx)bug14.out: $(gen-locales)
